### PR TITLE
fix: remove paste to bio box

### PR DESCRIPTION
**Pull request by zhouhanseng**
Remove deprecated 'paste-to-bio' box.

image

### CODE REVIEW

**Review by septs — APPROVED** [review]

**Comment by septs** [comment]
@guanbinrui enter 1.16.3?

File: src/social-network-provider/facebook.com/UI/injectKnownIdentity.tsx [hunk 1]
```diff
@@ -52,6 +52,8 @@ export function PersonKnownAtFacebook(props: PersonKnownProps) {
     )
 }
 
+// Todo: we could remove it as it is unused since we no longer need to paste public key to bio.
```
[the remark this answers is not in the document]
Comment (zhouhanseng) [reply src/social-network-provider/facebook.com/UI/injectKnownIdentity.tsx:55]: Not a temporary disabled.

File: src/social-network-provider/twitter.com/ui/injectKnownIdentity.tsx [hunk 2]
```diff
@@ -33,7 +33,8 @@ export function PersonKnownAtTwitter(props: PersonKnownProps) {
         />
     )
 }
-
+// Todo: we could remove it as it is unused since we no longer need to paste public key to bio.
+// Todo: But the code itself may be valuable, so keep it temporarily. - zhouhancheng
```
Comment (Jack-Works) [inline src/social-network-provider/twitter.com/ui/injectKnownIdentity.tsx:37]: You can always find the code in the git history. No need to leave it here.

Comment (zhouhanseng) [reply src/social-network-provider/twitter.com/ui/injectKnownIdentity.tsx:37]: Okay

**Review by septs — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**